### DISCUSSION TITLE
Ability to sign in on Google/Subscription content

### DISCUSSION THREAD

**Discussion by natobsid** [plateaukao/einkbro#186] [Q&A]
Not sure whether this is the right place for what I'm asking, but not sure it's a bug, so will post here. Please feel free to redirect me...

I am using Einkbro on a Boyue Likebook P78 and in general it is phenomenal. However, one issue that stops me from viewing content as I would like is that I can't sign in to my subscriptions. For example, I subscribe to several newspapers, such as ElPais.es. Unfortunately their apps are not optimised for e-ink so it would be great to be able to use Einkbro to view them. However, on Einkbro, when I try to sign into my accounts, all I get is a blank screen.

If I try to sign into a Google account, it tells me I don't have cookies enabled—but I'm not sure if this is the case.

Finally, on a possibly related note to the signing in problem, Einkbro doesn't seem to deal well with pop-ups. If there is a pop-up on the site, I can never dismiss them and therefore can't view the page.

If it's something that I have missed, I would appreciate any help on these (related?) issues. If not, perhaps it could become a feature request.

Thanks in advance—it really is a fantastic app for those of us who prefer e-ink screens!

**Comment by plateaukao** [comment]
@natobsid 
Do you see the tab count button in dot lines? If so, it's in incoginito mode.
If not, you should be able to login Google account. I could login mine on all my Eink devices, including my Boyue Likebook too.
As for other login mechanism, some sites do need some hack to make it work. For web pop-ups, there are too many combinations taht I couldn't handle all of them. In these scenarios, please use other more functional browsers instead.

**Reply by natobsid** [reply]
Hi, apologies for such a late reply. You hit the nail on the head, I had inadvertently switched it into incognito mode—thank you! I should have guessed when I was scratching my head wondering where my browser history was... Thanks again, this makes all the difference. This browser is a real game changer in terms of my e-reader use!

**Comment by plateaukao** [comment]
@natobsid 
Glad to be of help. I will provide a dialog to explain the long press feature, so that others won’t be confused if accidentally trigger it.
